subprojects/instant-execution/src/integTest/groovy/org/gradle/instantexecution/InstantExecutionAndroidIntegrationTest.groovy

```diff
@@ -17,9 +17,10 @@
 package org.gradle.instantexecution
 
 import org.gradle.integtests.fixtures.TestResources
+import org.gradle.test.fixtures.file.LeaksFileHandles
 import org.junit.Rule
 
-
+@LeaksFileHandles
```

Comment:
What causes these? Can we stop it from happening?

Reply:
The instant execution team can hopefully fix it: https://gradle.slack.com/archives/CH43P3A10/p1574106997020900

Reply:
I wonder if we could capture what Adam said there in that comment or something. 

My problem with `@LeaksFileHandles` and friends is that its hard to notice when the problem is accidentally fixed. Adding a comment could help whoever bumps into this next.

Reply:
I added a comment.

Reply:
I also opened https://github.com/gradle/instant-execution/issues/120 for this.